layouts/partials/docs/toc.html

```diff
@@ -5,20 +5,35 @@ <h4 class="no-anchor">
         </h4>
         {{ template "toc" (dict "page" . "menu" .Site.Menus.aws) }}
     {{ else if hasPrefix .RelPermalink "/docs/" }}
-        <h4 class="no-anchor">
-            <a href="{{ relref . "/docs/quickstart" }}">Get Started</a>
+        
+        <!-- Get Started guides and installation -->
+        <h5>Getting Started</h5>
+        <h4 class="no-anchor -mt-1">
+            <a href="{{ relref . "/docs/get-started" }}">Overview</a>
         </h4>
-        {{ template "toc" (dict "page" . "menu" .Site.Menus.quickstart) }}
-
-        <h4 class="no-anchor">
-            <a href="{{ relref . "/docs/reference" }}">Reference</a>
+        {{ template "toc" (dict "page" . "menu" .Site.Menus.getstarted) }}
+        <h4 class="no-anchor -mt-1">
+            <a href="{{ relref . "/docs/install" }}">Download and Install</a>
         </h4>
-        {{ template "toc" (dict "page" . "menu" .Site.Menus.reference) }}
-
-        <h4 class="no-anchor">
-            <a href="{{ relref . "/docs/console" }}">Pulumi Cloud Console</a>
+        
+        <!-- Core concepts, supported clouds, comparison -->
+        <h5>Intro</h5>
+        {{ template "toc" (dict "page" . "menu" .Site.Menus.intro) }}
+        
+        <!-- Guides and Tutorials -->
+        <h5>Guides and Tutorials</h5>
+        {{ template "toc" (dict "page" . "menu" .Site.Menus.userguides) }}
+        
+        <!-- API, CLI, and glossary -->
+        <h5>Reference</h5>
+        <h4 class="no-anchor -mt-1">
+            <a href="{{ relref . "/docs/reference" }}">Overview</a>
```

Comment:
The content in `/docs/reference/_index.md` needs to be updated. Right now it's the old content for `reference`, which links to things about Learning Pulumi, Using Pulumi in Production or in a Team, etc. It needs to turn into a page about API Reference and Commands.